library/general/src/modules/Popup.rb

```diff
@@ -74,6 +74,8 @@ def popupLayoutInternalTypeWithLabel(headline, message, button_box, label, richt
       button_box = deep_copy(button_box)
       content = Empty()
 
+      body = richtext ? message : convert_to_plaintext(message)
```

Comment:
This always escapes plain text messages. But maybe some existing calls pass already escaped text, that would cause double escaping.

Could you grep the YaST sources whether there is a such case? It's probably OK to change it, but we should double check first...